libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -16,7 +16,8 @@
 #define POSCONTROL_ACCELERATION_MIN             50.0f   // minimum horizontal acceleration in cm/s/s - used for sanity checking acceleration in leash length calculation
 #define POSCONTROL_ACCEL_XY                     100.0f  // default horizontal acceleration in cm/s/s.  This is overwritten by waypoint and loiter controllers
 #define POSCONTROL_ACCEL_XY_MAX                 980.0f  // max horizontal acceleration in cm/s/s that the position velocity controller will ask from the lower accel controller
-#define POSCONTROL_STOPPING_DIST_Z_MAX          200.0f  // max stopping distance vertically   
+#define POSCONTROL_STOPPING_DIST_UP_MAX         300.0f  // max stopping distance vertically while climbing
+#define POSCONTROL_STOPPING_DIST_DOWN_MAX       200.0f  // max stopping distance vertically while descending
```

Comment:
Could you had the units? I assume that it is cm.
The following comment looks weird or misplaced , no?